packages/components/src/components/Text/Text.tsx

```diff
@@ -115,6 +121,11 @@ export const Text: FC<TextProps> = ({
   const style: TextStyle = {
     ...typography[typographyKey as keyof typeof typography],
     color,
+    textAlign: center ? 'center' : 'auto',
+  }
+
+  if (bold && variant === 'body') {
```

Comment:
Should be redundant from typing:
```suggestion
  if (bold) {
```

Reply:
Updated